migrate.md

```diff
@@ -41,16 +41,16 @@ If you created service instances before November 2018, then depending on the loc
 ## Migrating a service instance
 {: #migrate-task}
 
-If you want to learn more about the migration process before you begin, see the [{{site.data.keyword.cloud_notm}} migration documentation ![External link icon](../../icons/launch-glyph.svg "External link icon")](/docs/resources?topic=resources-migrate){: new_window}.
+If you want to learn more about the migration process before you begin, see the [{{site.data.keyword.cloud_notm}} migration documentation](/docs/resources?topic=resources-migrate){: external}.
 
-Only the person or group that created the instance or that has the appropriate privileges can migrate it. For more information, see [Required access for service instances](/docs/resources?topic=resources-migrate#required_access_instances){: new_window}.
+Only the person or group that created the instance or that has the appropriate privileges can migrate it. For more details, see [Required access for service instances](/docs/resources?topic=resources-migrate#required_access_instances){: external}.
 {: note}
 
 To migrate your service instance, complete these steps:
 
 1.  Determine which resource group you want to move the service instance to.
 
-    See [Best practices for organizing resource in a resource group ![External link icon](../../icons/launch-glyph.svg "External link icon")](/docs/resources?topic=resources-bp_resourcegroups) for tips.
+    See [Best practices for organizing resource in a resource group](/docs/resources?topic=resources-bp_resourcegroups) for tips.
```

Comment:
Should this have an `external` on it, Allen?  Same with a few that follow.

Reply:
Nope. It's in the doc app.

Reply:
But i see another one does, so I removed that too.